### DISCUSSION TITLE
PLN Lab Week in Lisbon

### DISCUSSION THREAD

**Discussion by bmann** [ipvm-wg/spec#5] [Community Call]
Lab Week and related events are happening in Lisbon, Oct 24th - Nov 2nd https://labweek22.plnetwork.io

Who will be in Lisbon / what dates do you plan to be there?

If possible do a day or half day of working sessions. What would be the best day for you?

Comments below, please!

**Comment by RangerMauve** [comment]
I'll likely be there for the [IPFS Camp](https://ipfs.camp/) dates (28-30) on behalf of the IPLD team.

I could probably get PL to send me for other days for the WASM stuff too if it makes stuff easier for scheduling. 😁
